start/cli/src/main/java/org/ethereum/beacon/command/PrintSpec.java

```diff
@@ -0,0 +1,21 @@
+package org.ethereum.beacon.command;
+
+import org.ethereum.beacon.emulator.config.ConfigBuilder;
+import org.ethereum.beacon.emulator.config.YamlPrinter;
+import org.ethereum.beacon.emulator.config.chainspec.Spec;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "spec",
+    description = "Prints beacon chain constants defined by the spec")
+public class PrintSpec implements Runnable {
+
+  @Override
+  public void run() {
+    Spec spec =
+        new ConfigBuilder<>(Spec.class)
+            .addYamlConfigFromResources("/config/spec-constants.yml")
+            .build();
```

Comment:
I don't like that it doesn't print current task merged spec. Do you think it's better for user to merge it he-self?

Reply:
In my opinion, if user sets up it's own constants he should be aware of those values. Btw, what do you think about printing all overridden spec constants right after the plan when simulation begins?

Reply:
> printing all overridden spec constants
cool!